### DISCUSSION TITLE
Build the last version

### DISCUSSION THREAD

**Discussion by matinkg** [ixjf/MSIRGB#221] [General]
Hi, I always use the static color and since in the last version (v2.3.0) we should make 8 static colors, I should always change them together and it's a little boring.
I just noticed that there is an option in the last non-released version called (SET FOR ALL COLOURS) but unfortunately this version is not released yet and I don't have the VS to build my own.

I just wanted you guys to build the latest version for me if possible.
thank you <3

**Comment by ixjf** [comment]
Yeah, sorry, haven't been able to work on this for a while. Here you go:

[MSIRGB-unreleased.zip](https://github.com/ixjf/MSIRGB/files/6566344/MSIRGB-unreleased.zip)

**Reply by matinkg** [reply]
Thank you :heart:
